cmd/options.go

```diff
@@ -254,11 +253,51 @@ func (c *snapshotterOptions) validate() error {
 	if err := c.compressionConfig.Validate(); err != nil {
 		return err
 	}
-
 	return c.etcdConnectionConfig.Validate()
 }
 
 // complete completes the config.
 func (c *snapshotterOptions) complete() {
 	c.snapstoreConfig.Complete()
 }
+
+type copierOptions struct {
+	sourceSnapStoreConfig *types.SnapstoreConfig
+	snapstoreConfig       *types.SnapstoreConfig
+	maxSnapshots          int
+	maxSnapshotAge        int
+}
+
+func newCopierOptions() *copierOptions {
+	sourceSnapStoreConfig := snapstore.NewSnapstoreConfig()
+	sourceSnapStoreConfig.IsSource = true
+	return &copierOptions{
+		sourceSnapStoreConfig: sourceSnapStoreConfig,
+		snapstoreConfig:       snapstore.NewSnapstoreConfig(),
+	}
+}
+
+func (c *copierOptions) addFlags(fs *flag.FlagSet) {
+	fs.IntVar(&c.maxSnapshots, "max-snapshots", -1, "copy the last N snapshots form the store sorted by date from newest to oldest")
+	fs.IntVar(&c.maxSnapshotAge, "max-snapshot-age", -1, "copy the last snapshots form the store not older than N days")
```

Comment:
Even though I proposed (or approved) this naming in my previous review, now I believe it would be more correct to talk about "backups" and not snapshots here, a backup being the collection of a single full snapshot and all incremental snapshots based on it.

```suggestion
	fs.IntVar(&c.maxBackups, "max-backups", -1, "copy the specified number of backups sorted by date from newest to oldest")
	fs.IntVar(&c.maxBackupAge, "max-backup-age", -1, "copy only the backups not older than the specified number of days")
```